phpunit.xml.dist

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <phpunit colors="true" bootstrap="vendor/autoload.php">
+    <php>
+        <!-- Disable E_USER_DEPRECATED until reaching compatibility with Symfony 3.0 -->
+        <!-- php -r 'echo -1 & ~E_USER_DEPRECATED;' -->
+        <ini name="error_reporting" value="-16385"/>
```

Comment:
Okay I won't change anything in this PR until we agree on one solution. Thinking about the possibilities again I vote for locking on `"symfony/dependency-injection": "~2.2,<2.7"` in `dev` only. This way no notices are silenced (also possible future ones), users can still use DoctrineBundle with latest Symfony version and tests pass. This option is the best compromise IMO.